frontend/scripts/build.sh

```diff
@@ -18,8 +18,8 @@ else
 fi
 popd
 nix-shell --run "rm -rf .psci_modules/ .pulp-cache/ node_modules/ bower_components/ output/"
-nix-shell --run "npm install"
+nix-shell --run "yarn install"
```

Comment:
@domenkozar To have something like this `#! nix-shell -j 4 -i bash -p stack git yarn` ?